File: Pset-1/mario-less.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{
    int n;
    // take the input
    do
    {
        n = get_int ("Height: ");
    } while (n < 1 || n > 8);
   
   
    for (int i = 0; i < n; i++)
    {
        // for the space
        for (int j = 0; j < n - i - 1; j++)
        {
            printf (" ");
        }
        // for the hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf ("#");
        }
        printf ("\n");
    }
}

File: Pset-6/mario-less.py
def main():
    height = get_int()
    for i in range(height):
        for _ in range(height - i - 1):
            print(" ", end="")
        for _ in range(i + 1):
            print("#", end="")
        print()

def get_int():
    while True:
        n = int(input("Height: "))
        if n > 0 and n < 9:
            break
    return n

main()
